subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/CompileOptions.java

```diff
@@ -60,11 +59,6 @@
 
     private ForkOptions forkOptions = new ForkOptions();
 
-    private boolean useDepend;
-
-    @SuppressWarnings("deprecation")
-    private DependOptions dependOptions = new DependOptions();
-
```

Comment:
According to the commit history, this doesn't seem to be the case. These methods seem to just be another public API we haven't yet deprecated.